runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -170,8 +171,16 @@ public Dimension render(Graphics2D graphics)
 
 			if (config.showHAValue() && item.getHaPrice() > 0)
 			{
+				int haPrice = item.getHaPrice();
+				final ItemPrice natureRunePrice = itemManager.getItemPriceAsync(ItemID.NATURE_RUNE);
```

Comment:
You should put this inside the if block, so that the price is only looked up when needed.